man/tallow.patterns.5.md

```diff
@@ -0,0 +1,91 @@
+## tallow.patterns
+
+Tallow pattern matching configuration files.
+
+
+## SYNOPSIS
+
+tallow(1) configures patterns to match journal entries and banning thresholds
+from JSON files.
+
+`/etc/tallow/*.json`
+`/usr/share/tallow/*.json`
+
+
+## DESCRIPTION
+
+tallow(1) configures patterns to match journal entries and banning thresholds
+from JSON files. Many JSON files can exist for logical grouping. The tallow(1)
+daemon will read all JSON files from configuration directories at startup.
+
+tallow(1) will operate with default pattern definitions
+in`/usr/share/tallow/*.json`. Users can add more patterns with their own JSON
+files under `/etc/tallow`. The default JSON files can be overridden by creating
+the same file under `/etc/tallow`.
+
+
+## FILE FORMAT 
+
+Pattern configuration files use the JavaScript Object Notation (JSON) format.
+
+The JSON needs to be two levels deep and all properties required. The root
+object is an array containing objects with a `filter` key and an `items` key. 
+
+* `filter` is a string that defines a field for filtering the journal file.
+  This helps make sure patterns are only matched to a subset of journal
+  entries. See systemd.journal-fields(7) for valid journal fields.
+
+* `items` is an array of objects that contains three elements: `ban`, `score`,
+  and `pattern`.
+
+   * `ban` is an integer that defines the number of seconds the originating IP
```

Comment:
@ahkok can you confirm this is the logic for ban and score?

Reply:
correct. `ban` is "immediate block this host for X seconds`.